tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+
```

Comment:
There's a bunch of superfluous new lines, make sure to clean them up and use https://github.com/CntoDev/triscord/blob/master/tox.ini as an example.